app/login/components/color-palettes/color-palette-types.tsx

```diff
@@ -219,9 +221,12 @@ const DivergentColorPaletteCreator = (props: ColorPaletteCreatorProps) => {
 };
 
 const CategoricalColorPaletteCreator = (props: ColorPaletteCreatorProps) => {
-  const { onAdd, onRemove, onUpdate, colorValues } = props;
+  const { onAdd, onRemove, onUpdate, colorValues, title } = props;
   const classes = useStyles();
 
+  const addColorCondition =
+    CATEGORICAL_MAX_ALLOWED_COLORS <= colorValues.length || title.length === 0;
```

Comment:
Nit: I think `disabled` would also work, `addColorCondition` sounds to me like it should be true to render the component and here it's used to disable it 🤔